### DISCUSSION TITLE
New release on Play store

### DISCUSSION THREAD

**Discussion by sarowarhosen03** [termux/termux-app#4001] [General]
Today we found that my Play store show me an update of termux but I install it from fdroid. I check that termux don't - [ ] have any recent releases ![Screenshot_20240609-090422.jpg](https://github.com/termux/termux-app/assets/98280301/04ce619f-f3c4-4db9-b062-df2e5d3e061f)

**Comment by skj-skj** [comment]
check the discussion here: https://github.com/termux/termux-app/discussions/4000

**Comment by codedhacker** [comment]
Hi as I am sure you will already know the 'Play Store Version' either didn't work or wasn't supported any longer, could you please clarify whether this is version is now viable thanks.
Daniel Beresford

**Reply by twaik** [reply]
It is not. https://github.com/termux/termux-app/discussions/4000
